clean/new_orleans_csc.py

```diff
@@ -0,0 +1,123 @@
+import sys
+sys.path.append('../')
+import pandas as pd
+from lib.path import data_file_path
+from lib.columns import clean_column_names
+from lib.clean import clean_dates, clean_names
+from lib.uid import gen_uid
+
+
+def clean_suspension(df):
+    df.loc[:, 'action_appealed'] = df.action.str.lower().str.strip()\
+        .str.replace('*', '', regex=False)\
+        .str.replace(r'\(|\)', '', regex=True)\
+        .str.replace('--', '-', regex=False)\
+        .str.replace(r'(\w+) & (\w+)', r'\1/\2', regex=True)\
+        .str.replace(r'sus?pension?n?', 'suspension', regex=True)\
+        .str.replace(r'suspended days (\d+)', r'\1-day suspension', regex=True)\
+        .str.replace(r'suspension-(\w+)', r'suspension/\1', regex=True)\
+        .str.replace(r'(\d+) day', r'\1-day', regex=True)\
+        .str.replace(r'(\d+) months', r'\1-months', regex=True)\
+        .str.replace('suspended', 'suspension', regex=False)\
+        .str.replace('suspension from working details for 60-day', 
+                     '60-day suspension from working detail', regex=False)\
+        .str.replace('detail privileges suspension for 12-months', 
+                     '12-month suspension from detail privileges', regex=False)\
+        .str.replace('suspension working details for 64-days', 
+                     '64-day suspension from working detail', regex=False)\
+        .str.replace('lwop', 'leave without pay', regex=False)\
+        .str.replace('other', '', regex=False)
+    return df.drop(columns='action')
+
+
+def clean_on_docket(df):
+    df.loc[:, 'motions'] = df.on_docket.str.lower().str.strip()\
+        .str.replace(r'(\w+)---?(\w+)', r'\1-\2', regex=True)\
+        .str.replace(r'(\w+) - (\w+)', r'\1-\2', regex=True)\
+        .str.replace('motion / pteition for nullity', 'motion to petition for nullity', regex=False)\
+        .str.replace(r', (\w+)', r'/\1', regex=True)\
+        .str.replace(r'(\w+)   ? ? ?(\w+)', r'\1 \2', regex=True)\
```

Comment:
Perhaps this will be shorter? `.str.replace(r'(\w+) +(\w+)', r'\1 \2', regex=True)\`